### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Viktor535x**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,14 +4,121 @@
...
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="/action_page.php" autocomplete ="off">
+      <fieldset enabled>
+        <legend>Personal information</legend>
+        <label class="lab_inp">Surname:
+          <input type="text" id="fname"
+            name="camelCase" required
+          >
```
Comment (ivanchuk-yurii) [inline src/index.html:22]:
```suggestion
        <label class="lab_inp">
          Surname:
          <input
            type="text"
            id="fname"
            name="camelCase"
            required
          >
```
fix in other places

File: src/index.html [hunk 2]
```diff
@@ -4,14 +4,121 @@
...
+      <fieldset enabled>
+        <legend>Registration</legend>
+        <label class="lab_inp">E-mail:
+          <input type="email" id="email"
+            name="camelCase" placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="lab">Password:
+          <input type="Password" id="Password"
```
Comment (ivanchuk-yurii) [inline src/index.html:57]:
```suggestion
          <input type="password" id="Password"
```

File: src/index.html [hunk 3]
```diff
@@ -4,14 +4,121 @@
...
+           name="camelCase" required
+          >
+        </label>
+      </fieldset>
+      <fieldset enabled>
+        <legend>An interesting fact about you!
+        </legend>
+        <li class="lab_inp">
```
Comment (ivanchuk-yurii) [inline src/index.html:65]: don't use `li` without `ul/ol`

File: src/style.css [hunk 4]
```diff
@@ -1 +1,12 @@
-/* styles go here */
+fieldset {
```
Comment (ivanchuk-yurii) [inline src/style.css:1]: don't style tags

File: src/style.css [hunk 5]
```diff
@@ -1 +1,12 @@
...
+  margin-bottom: 20px;
+}
+
+.lab_inp {
```
Comment (ivanchuk-yurii) [inline src/style.css:5]: use full names for classes

File: src/index.html [hunk 6]
```diff
@@ -4,14 +4,121 @@
...
+          </select>
+        </label>
+        <label class="lab">How do you rate our work?
+          <input type="range" id="rate"
+            name="camelCase" min="100"
+            max="100"
+          >
+        </label>
+      </fieldset>
+      <fieldset enabled>
```
Comment (ivanchuk-yurii) [inline src/index.html:105]: what is `enabled` for?

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

**Review by Bohdan-Maksymiuk — COMMENTED** [review]
PR is closed - open it so we will able to check it

**Pull request closed without merging**